test/functional/fixtures/ui/testcafe-fixtures/status-bar-test.js

```diff
@@ -0,0 +1,101 @@
+import { Selector } from 'testcafe';
+import OS from 'os-family';
+import { saveWindowState, restoreWindowState } from '../../../window-helpers';
+
+fixture `Status Bar`
+    .page `http://localhost:3000/fixtures/ui/pages/empty-page.html`
+    .beforeEach(async t => {
+        await saveWindowState(t);
+    })
+    .afterEach(async t => {
+        await restoreWindowState(t);
+    });
+
+test('Show status prefix', async t => {
+    const statusDiv = Selector(() => window['%testCafeDriverInstance%'].statusBar.statusDiv);
+
+    let statusText = await statusDiv.innerText;
+
+    await t
+        .expect(statusText).notOk()
+        .expect(statusDiv.innerText).eql('Waiting for assertion execution...');
+
+    await t
+        .eval(() => window['%testCafeDriverInstance%'].statusBar.setStatusPrefix('Status prefix'));
+
+    statusText = await statusDiv.innerText;
+
+    await t
+        .expect(statusText.trim()).eql('Status prefix.')
+        .expect(statusDiv.innerText).eql('Status prefix. Waiting for assertion execution...')
+        .navigateTo('about:blank')
+        .expect(statusDiv.innerText).eql('Status prefix. Waiting for assertion execution...');
+});
+
+test('Hide elements when resizing the window', async t => {
+    const statusBarDiv = Selector(() => window['%testCafeDriverInstance%'].statusBar.statusBar);
+    const statusDiv    = Selector(() => window['%testCafeDriverInstance%'].statusBar.statusDiv);
```

Comment:
(minor) let's exract statusBar selectors, place them in the begging of the test file and reuse them